packages/c/sshnpd/src/params.c

```diff
@@ -179,5 +179,11 @@ int parse_sshnpd_params(sshnpd_params *params, int argc, const char **argv) {
   }
 
   // Repeat for permit-open
+  // Convert devicename to lower case
+  for (char *c = params->device; c[0] != '\0'; c++) {
```

Comment:
This reads: start the pointer at the beginning of the device name, move it forward once each iteration until we reach the null terminator.

Reply:
This string is expected to always be null-terminated, unless a bug occurs in argparse, or we change how our defaults are set.